doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+Learn more about our [doughnut charts](../doughnut-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Area Chart
+
+Learn more about our [area charts](chart-types-area.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### Sparkline Chart 
+
+A Sparkline Chart, or Sparkline Graph is a type of category graph intended for rendering within a small-scale layout such as within a grid cell, or anywhere a word-sized visualization is needed to tell a data story. Like other $PlatformShort$ chart types, the Sparkline Chart has several visual elements and corresponding features that can be configured and customized such as the chart type, markers, ranges, trendlines, unknown value plotting, and tooltips. Sparkline charts can render as a Line Chart, Area Chart, Column Chart or Win / Loss Chart. The difference between the full-sized chart equivalent to the Spark-chart, is the Y-Axis (left side labels) and X-Axis (bottom labels) are not visible. Learn more about our [sparkline charts](../sparkline.md).
+
+### Bubble chart 
+
+A Bubble Chart, or Bubble Graph, is used to show data comprising of three numeric values. Two of the values are plotted as an intersecting point using a Cartesian (X, Y) coordinate system, and the third value is rendered as the diameter size of the point. This gives the Bubble Chart its name - a visualization of varying sized bubbles along the X and Y coordinates of the plot. The $PlatformShort$ Bubble Chart is used to show relationships of data correlations with the data value differences rendered by size. You can also use a fourth data dimension, typically color, to further differentiate the values in your Bubble chart. Learn more about our [bubble charts](chart-types-scatter.md#$PlatformShort$-Scatter-Bubble-Chart).
+
+### Financial chart / Stock Chart 
+
+The $PlatformShort$ Financial or Stock Chart, is a composite visualization that renders stock data and financial data in a time-series chart that includes interactive visual elements in a toolbar like day / week / month filters, chart type selection, volume type selection, indicators selection and trends lines selection. Designed for customization, the $PlatformShort$ Stock Chart can be customized in any way to give an easier visualization and interpretation of your data. The financial chart renders the date-time data along the X-Axis (bottom labels) and shows fields like Open, High, Low and Close volumes. The type of chart to render the Time-Series data can be Bar, Candle, Column, or Line. Learn more about our [financial charts](chart-types-financial.md).
+
```

Comment:
https://share.goabstract.com/f7dde8dc-e79b-4428-91f9-5cc512c17070

I still haven't got data for this sample, but I guess we can use a data for some of the stock indexes like S&P 500 e.g.

Reply:
Found the data.

[S&P500 PerformanceGraphExport 3 years.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5903041/S.P500.PerformanceGraphExport.3.years.xlsx)